post_coverage/test.rb

```diff
@@ -0,0 +1,5 @@
+require 'octokit'
+
```

Comment:
### Incremental code coverage report 
New code lines here are not covered by tests, please add tests on marked lines.